src/components/standalone/backup_and_restore/RestoreContent.vue

```diff
@@ -186,9 +186,26 @@ async function restoreBackup() {
         errorRestoreBackup.value.notificationTitle = t('error.cannot_restore_backup')
       }
     } catch (exception: any) {
-      errorPage.value.notificationTitle = t('error.cannot_restore_backup')
-      errorPage.value.notificationDescription = t(getAxiosErrorMessage(exception))
-      errorPage.value.notificationDetails = exception.toString()
+      if (exception instanceof ValidationError) {
```

Comment:
- Validation errors (e.g. invalid passphrase) should be displayed with the red message under the invalid field, without showing the error notification. Runtime errors (e.g. restore failed) should be displayed only with the error notification.
- The validation error can be managed like this: https://github.com/NethServer/nethsecurity-ui/blob/main/src/components/standalone/firewall/nat/CreateOrEditNetmapRuleDrawer.vue#L268-L273
- Let's display under the passhprase field "Incorrect passphrase or invalid backup file" (use this string for the i18n keyfor "decryption_failed").